### PR TITLE
Screen Alarms for EVERYONE!

### PR DESCRIPTION
**Pull request by jknpj**
How the new HUDs look with alarms:
vampire

Now we can finally add more alarms without fucking up the old HUD elements.

:cl:
 * rscadd: Converted every alarm HUD to screen alarms
 * rscadd: Non-silicons will now receive the buckle screen alarm

### CODE REVIEW

**Comment by jknpj** [comment]
I need to add some screenshots later but for most mobs the health HUD was moved down a little(soul sword health HUD is now under the shade health HUD) and the Plasma/Genes/Blood counter is now under the health HUD.

File: code/_onclick/hud/screen_alarms.dm [hunk 1]
```diff
@@ -160,9 +174,101 @@ var/global/list/screen_alarms_locs = list(
 
 /obj/abstract/screen/alert/object/buckled
 	name = "Buckled"
-	desc = "You've been buckled to something and can't move. Click this alert to unbuckle unless you're unable to."
+	desc = "You've been buckled to something and can't move. Click on this alert to unbuckle."
+
+//Carbon Alarms
+/obj/abstract/screen/alert/carbon/breath
+	name = "Choking"
```
Comment (Kurfursten) [inline code/_onclick/hud/screen_alarms.dm]: technically, this is really suffocating, choking implies something obstructing the airway

**Comment by Kurfursten** [comment]
There's so much pasting for various alerts, like body temperature for instance. Is there no way we can move this up to a higher level?

**Comment by jknpj** [comment]
Life() is a mess.

**Comment by jknpj** [comment]
@Kurfursten You mean the silicon/carbon alerts?

**Comment by Kurfursten** [comment]
Alien, complex, human, monkey, etc they all have a big switch statement for temperatures/pressures to throw alarms

Can't we just move this up to living level?

**Comment by jknpj** [comment]
If only you knew how every subtype of alien has a copypaste of old human code

**Comment by Kurfursten** [comment]
You don't have to migrate all of it, but if you could just *start* the process of inheritance by taking the step here [just for alarms], that would be good

**Comment by jknpj** [comment]
I'd rather do it in another PR because this one is supposed to just convert the hardcoded hud alarms into screen alarms

**Pull request merged**